securedrop_client/gui/widgets.py

```diff
@@ -1552,18 +1553,70 @@ class ReplyWidget(SpeechBubble):
     Represents a reply to a source.
     """
 
-    CSS_COLOR_BAR_REPLY_FAIL = '''
-    background-color: #ff3366;
+    CSS_REPLY_FAILED = '''
+    #message {
+        min-width: 540px;
+        max-width: 540px;
+        font-family: 'Source Sans Pro';
+        font-weight: 400;
+        font-size: 15px;
+        background-color: #fff;
+        color: #3b3b3b;
+        padding: 16px;
+    }
+    #color_bar {
+        min-height: 5px;
+        max-height: 5px;
+        background-color: #ff3366;
+        border: 0px;
+    }
     '''
 
-    CSS_COLOR_BAR_REPLY = '''
-    background-color: #0065db;
+    CSS_REPLY_SUCCEEDED = '''
+    #message {
+        min-width: 540px;
+        max-width: 540px;
+        font-family: 'Source Sans Pro';
+        font-weight: 400;
+        font-size: 15px;
+        background-color: #fff;
+        color: #3b3b3b;
+        padding: 16px;
+    }
+    #color_bar {
+        min-height: 5px;
+        max-height: 5px;
+        background-color: #0065db;
+        border: 0px;
+    }
+    '''
+
+    # Custom pending CSS styling simulates the effect of opacity which is only
+    # supported by tooltips for QSS.
+    CSS_REPLY_PENDING = '''
```

Comment:
can we change the opacity for the bar too? particularly when the speech bubble is retrying the difference in opacity is noticeable, took a screenshot in case it helps:
![Screenshot_2019-10-29_18-08-35](https://user-images.githubusercontent.com/4522213/67813893-ef970600-fa5f-11e9-860c-f0855b6360d0.png)

Reply:
Sounds good. I'll add it to the list of things to discuss next UI/UX revie.

Reply:
Should we nominate this to discuss in tomorrow's UX meeting?

Reply:
TBH, I am confused as to why the replies in the prior session are blue, but the replies from the current session are in purple. If we don't show journalist badges with the replies themselves, the user won't know why, either. I also want to ensure that if the prior replies to a Source were from the journalist that's currently logged-on, that they're seeing those replies in purple, too. 

If we're not yet supporting multi-journalist color coding, then ALL replies successfully sent to a Source, should be the same color; regardless of who's logged in.

Reply:
we haven't implemented showing journalist badges yet in the conversation view, nor the per-journalist colors, so right now the messages from all journalists are blue. I can change the pending color bar to also be blue

Reply:
> can we change the opacity for the bar too? 

we discussed this today in a ux sync and nina pointed out that the bar with full opacity stands out less once we add journalist icons next to replies, so I'm resolving this conversation